test/integration/maincluster_test.go

```diff
@@ -3475,7 +3475,7 @@ worker:
 			configBytes := validCase.configYaml
 			// TODO Allow including plugins in test data?
 			plugins := []*pluginmodel.Plugin{}
-			providedConfig, err := config.ConfigFromBytesWithEncryptService([]byte(configBytes), plugins, helper.DummyEncryptService{})
+			providedConfig, err := config.ConfigFromBytesWithMocks([]byte(configBytes), plugins, helper.DummyEncryptService{}, helper.DummyCFInterrogator{})
```

Comment:
Nit but it sounds more like "stubs" rather than `mocks`. In my understanding stubs are these simpler alternative objects that are used to replace the complex aws client in order to test the `providedConfig` in isolation. Mocks on the other hand would be used to verify how methods are called, which doesn't seem to be what we're doing here.